src/ansys/pyensight/core/utils/omniverse_dsg_server.py

```diff
@@ -770,6 +806,13 @@ def end_update(self) -> None:
         default=False,
         help="In this mode do not include a camera or the case level matrix.  Geometry only.",
     )
+    parser.add_argument(

```

Comment:
Do we need this in the repo, especially if:   'if args.debugWait:' is just 
'pass'?

Reply:
@david-bremer let's push this in and I will update this when I merge in my PR